jobs/service-fabrik-backup-manager/templates/config/settings.yml.erb

```diff
@@ -54,6 +54,10 @@ production:
   apiserver:
     ip: <%= link("service-fabrik-apiserver").p('ip') %>
     port: <%= link("service-fabrik-apiserver").p('port') %>
+    ca: <%= JSON.dump(link("service-fabrik-apiserver").p('tls.apiserver.ca')) %>
+    certificate: <%= JSON.dump(link("service-fabrik-apiserver").p('tls.apiserver.certificate')) %>
+    private_key: <%= JSON.dump(link("service-fabrik-apiserver").p('tls.apiserver.private_key')) %>
```

Comment:
can we have property keys like **cert, key** for consistency ?

Reply:
ok